my_class.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'my_enumerable'
```

Comment:
- [ ] You have a typo here. The file named `my_enumerables` is not `my_enumerable`, kindly consider renaming the file to change the import statement to `require_relative 'my_enumerables'